scripts/lint-staged/eslint.js

```diff
@@ -94,7 +103,11 @@ async function runEslintOnFilesGroupedPerPackage() {
     });
 }
 
-runEslintOnFilesGroupedPerPackage().catch(err => {
-  console.error(err);
-  process.exit(1);
-});
+function main() {
```

Comment:
to follow our standard script pattern I wrapped the logic within `main` function